crates/bevy_ecs/src/schedule/set.rs

```diff
@@ -157,3 +186,39 @@ where
         SystemTypeSet::new()
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::{
+        collections::hash_map::DefaultHasher,
+        hash::{Hash, Hasher},
+    };
+
+    use super::AnonymousSet;
+
+    #[test]
+    fn same_instance() {
```

Comment:
I don't think we need this test anymore, as this is just testing the auto-impled hasher.

Reply:
Removed.